src/PromptEngineeringSidebar.tsx

```diff
@@ -166,71 +165,104 @@ function PromptEngineeringSidebar(props: {
             label="Use a system message template"
             onChange={handleChangeTemplate}
           >
-            {templates.map((template) => {
-              return (
-                <MenuItem value={template.id} key={template.id}>
-                  {template.humanReadableName}
-                </MenuItem>
-              );
-            })}
+            {templates.map((template) => (
+              <MenuItem key={template.id} value={template.id}>
+                {template.humanReadableName}
+              </MenuItem>
+            ))}
           </Select>
         </FormControl>
+
         <TextField
           label="System message"
           multiline
           rows={4}
           value={props.systemPrompt}
           onChange={(e) => props.changeSystemPrompt(e.target.value)}
         />
+
         <Button
           variant="outlined"
           startIcon={<AddIcon />}
-          onClick={handleAddExample}
+          onClick={() =>
+            props.changeExamples([
+              ...props.examples,
+              { user: "", assistant: "" },
+            ])
+          }
         >
           Add example
         </Button>
-        {props.examples.map((example, idx) => {
-          return (
-            <Accordion key={idx}>
-              <AccordionSummary
-                expandIcon={<ExpandMoreIcon />}
-                aria-controls="panel1-content"
-                id="panel1-header"
+
+        {props.examples.map((example, idx) => (
+          <Accordion key={idx}>
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon />}
+              aria-controls="panel1-content"
```

Comment:
we should include the key in the id and aria-controls

Reply:
included